orchagent/portsorch.cpp

```diff
@@ -2955,6 +2959,22 @@ bool PortsOrch::initializePort(Port &port)
 
     initializePriorityGroups(port);
     initializeQueues(port);
+    if (m_isQueueMapGenerated)
```

Comment:
Again, I think it is good to have a per port flag, that we could check here before we re-generate,  also after we done with generation, we set the per port flag.

Reply:
Sure. But note that the if condition to call generateQueueMapPerPort and generatePriorityGroupMapPerPort will be 
```
if (m_isQueueMapGenerated && !port->m_isQueueMapGenerated)
```
and
```
if (m_isPriorityGroupMapGenerated && !port->m_isPriorityGroupMapGenerated)
```

respectively.